powpeg-details.js

```diff
@@ -21,7 +21,7 @@ class PowpegDetails {
     }
 }
 
-module.exports = async (web3, networkSettings) => {
+module.exports = async (web3, network, networkSettings) => {
```

Comment:
Could we get the network from the networkSettings? I feel that would make it simpler and coherent, since the the networkSettings are related to a specific network

Reply:
The networkSettings Currently Doesn't Have Or Returns Anything Related To The Specific Network